test/jdk/java/awt/print/PrinterJob/PrintImage.java

```diff
@@ -21,276 +21,167 @@
  * questions.
  */
 
-/**
+import java.awt.BorderLayout;
+import java.awt.Canvas;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Frame;
+import java.awt.Graphics;
+import java.awt.Menu;
+import java.awt.MenuBar;
+import java.awt.MenuItem;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterException;
+import java.awt.print.PrinterJob;
+
+import jtreg.SkippedException;
+
+/*
  * @test %I %W
```

Comment:
```suggestion
 * @test
```
These should be removed.